cmake/BuildLibElf.cmake

```diff
@@ -0,0 +1,15 @@
+include(ExternalProject)
+
+ExternalProject_Add(libelf
+  PREFIX "vendor/elfutils"
+  URL http://archive.ubuntu.com/ubuntu/pool/main/e/elfutils/elfutils_0.163.orig.tar.bz2
```

Comment:
Ouch! Well, that settles it I guess.

@drzaeus77: What would you prefer? Runtime dependency on `libelf.so`?